File: CS50-Cash-Python/cash.py
from cs50 import get_float

change = -1
while change < 0:
    change = get_float("Change owed: ")

change = change*100

coins = 0
while change > 0:
    if change >= 25:
        coins += 1
        change -= 25
        continue
    elif change >= 10:
        coins += 1
        change -= 10
        continue
    elif change >= 5:
        coins += 1
        change -= 5
        continue
    elif change >= 1:
        coins += 1
        change -= 1

print(coins)

File: CS50-Cash-C/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float change;
    do
    {
        change = get_float("Change owed: ");
    }
    while (change < 0);

    int cents = round(change * 100);
    int rem_change = cents;
    int coins_used = 0;

    while (rem_change > 0) {
        if (rem_change >= 25) {
            rem_change = rem_change - 25;
            coins_used++;
        } else if (rem_change >= 10) {
            rem_change = rem_change - 10;
            coins_used++;
        } else if (rem_change >= 5) {
            rem_change = rem_change - 5;
            coins_used++;
        } else if (rem_change >= 1) {
            rem_change = rem_change - 1;
            coins_used++;
        }     
    }
    printf("%i \n", coins_used);
}
